.github/workflows/dotnet-build.yml

```diff
@@ -138,8 +150,47 @@ jobs:
       - name: Display dotnet info
         run: dotnet --info
 
-      - name: Build CommunityToolkit.Maui
-        run: dotnet build -c Release ${{ env.PathToLibrarySolution }}
+      - name: 'Build CommunityToolkit.Maui.Analyzers'
+        run: dotnet build ${{ env.PathToCommunityToolkitAnalyzersCsproj }} -c Release
+
+      - name: 'Build CommunityToolkit.Maui.Analyzers.CodeFixes'
+        run: dotnet build ${{ env.PathToCommunityToolkitAnalyzersCodeFixCsproj }} -c Release
+
+      - name: 'Build CommunityToolkit.Maui.Camera.Analyzers'
+        run: dotnet build ${{ env.PathToCommunityToolkitCameraAnalyzersCsproj }} -c Release
+
+      - name: 'Build CommunityToolkit.Maui.Camera.Analyzers.CodeFixes'
+        run: dotnet build ${{ env.PathToCommunityToolkitCameraAnalyzersCodeFixCsproj }} -c Release
+
+      - name: 'Build CommunityToolkit.Maui.MediaElement.Analyzers'
+        run: dotnet build ${{ env.PathToCommunityToolkitMediaElementAnalyzersCsproj }} -c Release
+
+      - name: 'Build CommunityToolkit.Maui.MediaElement.Analyzers.CodeFixes'
+        run: dotnet build ${{ env.PathToCommunityToolkitMediaElementAnalyzersCodeFixCsproj }} -c Release
+
+      - name: 'Build CommunityToolkit.Maui.SourceGenerators'
+        run: dotnet build ${{ env.PathToCommunityToolkitSourceGeneratorsCsproj }} -c Release
+
+      - name: 'Build CommunityToolkit.SourceGenerators.Internal'
+        run: dotnet build ${{ env.PathToCommunityToolkitSourceGeneratorsInternalCsproj }} -c Release
+
+      - name: 'Build CommunityToolkit.Maui.Camera'
+        run: dotnet build ${{ env.PathToCommunityToolkitCameraCsproj }} -c Release -p:PackageVersion=${{ env.NugetPackageVersionCamera }} -p:Version=${{ env.NugetPackageVersion }}
+
+      - name: 'Build CommunityToolkit.Maui.MediaElement'
+        run: dotnet build ${{ env.PathToCommunityToolkitMediaElementCsproj }} -c Release -p:PackageVersion=${{ env.NugetPackageVersionMediaElement }} -p:Version=${{ env.NugetPackageVersion }}
+
+      - name: 'Build CommunityToolkit.Maui.Maps'
+        run: dotnet build ${{ env.PathToCommunityToolkitMapsCsproj }} -c Release -p:PackageVersion=${{ env.NugetPackageVersionMaps }} -p:Version=${{ env.NugetPackageVersion }}
+
+      - name: 'Build CommunityToolkit.Maui.Core'
+        run: dotnet build ${{ env.PathToCommunityToolkitCoreCsproj }} -c Release -p:PackageVersion=${{ env.NugetPackageVersion }} -p:Version=${{ env.NugetPackageVersion }}
+
+      - name: 'Build CommunityToolkit.Maui'
+        run: dotnet build ${{ env.PathToCommunityToolkitCsproj }} -c Release -p:PackageVersion=${{ env.NugetPackageVersion }} -p:Version=${{ env.NugetPackageVersion }}
+
+      - name: 'Run CommunityToolkit.Maui.Analyzers.UnitTests'
+        run: 'dotnet test ${{ env.PathToCommunityToolkitAnalyzersUnitTestCsproj }} -c Release'
```

Comment:
We run test for the whole solution. This can be removed

Reply:
Ah - good catch! Removed ✔️